src/dioptra/restapi/v1/workflows/service.py

```diff
@@ -64,3 +83,87 @@ def get(self, job_id: int, file_type: FileTypes, **kwargs) -> IO[bytes]:
             file_type=file_type,
             logger=log,
         )
+
+
+class EntrypointValidateService(object):
+    """"""
+
+    @inject
+    def __init__(
+        self,
+        plugin_id_service: PluginIdsService,
+        plugin_id_file_service: PluginIdFileService,
+    ) -> None:
+        """Initialize the entrypoint service.
+
+        All arguments are provided via dependency injection.
+
+        Args:
+            plugin_ids_service: A PluginIdsService object.
+        """
+        self._plugin_id_service = plugin_id_service
+        self._plugin_id_file_service = plugin_id_file_service
+
+    def validate(
+        self, 
+        task_graph: str, 
+        plugin_ids: list[int], 
+        entrypoint_parameters: list[dict[str, Any]],
+        **kwargs,
+    ) -> dict[str, Any]:
+        """Validate a entrypoint workflow before the entrypoint is created.
+
+        Args:
+            task_graph:
+            plugin_ids: 
+            parameters: 
+
+        Returns:
+            
+
+        Raises:
+            
+        """
+        log: BoundLogger = kwargs.get("log", LOGGER.new())
+        log.debug("Validate a entrypoint workflow", task_graph=task_graph, plugin_ids=plugin_ids, entrypoint_parameters=entrypoint_parameters)
+
+        parameters = {param['name']: param['default_value'] for param in entrypoint_parameters}
+
+        tasks: dict[str, Any] = {}
```

Comment:
I think we should move this block (line 132 to 161) to `export_task_engine_yaml.py`.  It would be a function similar to `export_task_engine_yaml` but with different inputs.